src/main/java/com/opencart/datamodel/RegisterModel.java

```diff
@@ -0,0 +1,15 @@
+package com.opencart.datamodel;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
```

Comment:
Please, for better understanding, try to implement builder without the annotation. In case, if you will have some troubles, you can see implementation in my repository

Reply:
we shouldn't use the lombok at all or  just  in this case?

Reply:
Just in this case, I want you to understand how the builder really works. If somebody on interview asked you ”What do you know about builder?”, you would answer on that without any problems) Because, lombok hidden the implementation